### DISCUSSION TITLE
OpenJDK 23 GA

### DISCUSSION THREAD

**Discussion by hi-Das** [microsoft/openjdk#630] [Q&A] [answered]
I see that openjdk 23 GA is available. https://jdk.java.net/23/
When can we expect the Microsoft version of OpenJDK 23?

**Comment by brianjstafford** [comment, marked as answer]
Microsoft only releases Long Term Support (LTS) versions of OpenJDK, which currently includes JDK 21, 17, and 11. Our next LTS version of OpenJDK will be JDK 25, to be released in September 2025.
